### PR TITLE
bump assets-controllers to v9.1.0

### PR DESCRIPTION
**Pull request by adonesky1**
Fast follow to https://github.com/MetaMask/metamask-extension/pull/19454

This adds some fixes to NFTController to ensure that if the network changes before the user accepts a `wallet_watchAsset` NFT request, the NFT is added to the chain ID and address before the request was initiated.

No breaking changes were introduced in this release so no adaptations are required.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -229,7 +229,7 @@
     "@metamask/address-book-controller": "^3.0.0",
     "@metamask/announcement-controller": "^4.0.0",
     "@metamask/approval-controller": "^3.1.0",
-    "@metamask/assets-controllers": "^9.0.0",
+    "@metamask/assets-controllers": "^v9.1.0",
```
[the remark this answers is not in the document]
Comment (adonesky1) [reply package.json]: done [here](https://github.com/MetaMask/metamask-extension/pull/19629/commits/4899aaef4a725a58b18c191f10dd9c5569b769d5)

**Comment by mcmire** [comment]
@adonesky1 It does appear that way, yes. `mockttp` isn't being updated here.

**Comment by adonesky1** [comment]
@SocketSecurity ignore @httptoolkit/subscriptions-transport-ws@0.9.19

**Review by mcmire — APPROVED** [review]

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**